api/src/pcapi/scripts/fill_offer_address/main.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+import logging
+import time
+
+import sqlalchemy as sa
+
+from pcapi.app import app
+from pcapi.models import db
+
+
+app.app_context().push()
+logger = logging.getLogger(__name__)
+
+
+def do_all_in_sql(min_id: int, max_id: int, batch_size: int, dry_run: bool, do_vacuum: bool = False) -> None:
+    for i in range(min_id, max_id, batch_size):
+        db.session.execute(
+            sa.text(
+                """
+            UPDATE offer SET "offererAddressId" = p."offererAddressId" from (
+            select venue."offererAddressId", offer.id from offer join venue on offer."venueId" = venue.id where offer."offererAddressId" is null 
+            and  (offer.url = '' or offer.url is NULL) 
+            and offer.id >= :min_id 
+            and offer.id <:max_id 
+            ) as p 
+            where offer.id = p.id
+            and offer.id >= :min_id 
+            and offer.id <:max_id ;            
```

Comment:
Est-ce que l'on a vraiment besoin d'avoir ces conditions (`offer.id >= :min_id AND offer.id <:max_id`) ici en plus de les avoir dans la sous requête ?